src/styles/index.scss

```diff
@@ -1,3 +1,8 @@
 body {
   margin: 0;
+  padding: 0;
 }
+
+@import 'variables';
```

Comment:
Ensure that the `variables.scss` file contains all the necessary variable definitions used in your SCSS files, such as `$white`, `$gray-elements`, `$main-accent`, `$secondary`, and `$blue-accent`.